stack.yaml

```diff
@@ -1,29 +1,9 @@
-resolver: lts-13.26
+resolver: lts-14.5
```

Comment:
I need to update docker image first for this PR to be merged in `master`. Will do it soon